docs/source/reference/local/index.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
If we decide users should use older versions on readthedocs, may as well remove local/*?

Reply:
Good point, removing local/*.